ignore errors in stats package

tbg opened the pull request:
the errors returned here are trivial (empty slice)
and we panicked on them. this update uses
the zero value when such an error is expected and
skips the error otherwise.

tamird commented on kv/txn_coord_sender.go:
```diff
@@ -277,37 +277,24 @@ func (tc *TxnCoordSender) startStats() {
 			num := len(curStats.durations)
-			dMax, err := stats.Max(curStats.durations)
-			if err != nil {
-				panic(err)
```
I don't think we should be ignoring these; the error semantics may change in a future version of `stats`, and the panics are harmless if we're never hitting them in practice. Other changes LGTM

tschottdorf replied:
done.

tbg committed:
ignore errors in stats package

the errors returned here are trivial (empty slice)
and we panicked on any of them. this update uses
the zero value when such an error is expected and
skips the error otherwise.
